package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "@maxmind/geoip2-node": "^3.1.0",
     "@sentry/node": "5.27.2",
+    "axios": "^0.27.2",
```

Comment:
Interesting, I wasn't aware that had landed yet.
https://dev.to/cloudx/nodejs-18-fetch-api-test-runner-module-and-more-2ckg

But we'd (global we, not me) also have to check all the current usages of **got** and see how difficult it'd be to migrate to a native **fetch** implementation (since I'm sure **got** and **axios** have a bunch of nice helpers built-in which might need to be recreated manually if using **fetch**). 
Also, https://nodejs.org/api/globals.html#fetch lists the fetch API (added in 17.5.0) as:
> _"**Stability: 1** - Experimental. Disable this API with the [`--no-experimental-fetch`](https://nodejs.org/api/cli.html#--no-experimental-fetch) CLI flag."_

&hellip; which triggers my spidey-senses. ⚡🕷️⚡

Reply:
will look to refactor using got